### ISSUE TITLE
Unable to install addon in anki 

### ISSUE THREAD

**Issue by sp3ctum** [Ben-Kerman/anki-jrp#4]
Hi, I was a user of the MIA/migaku tools for a couple of years until about a week ago when I updated my Linux Mint system and my old Anki could not start up any longer.

I came looking for an alternative and was pointed to your project - it looks very promising! But I am having an issue installing the addon.

I downloaded `japanese-readings-and-pitch-accent_0.1_ipadic.ankiaddon` and when I try to install it in anki's update manager I get this error:

```
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue on the add-on support site.
Debug info:
Anki 2.1.49 (dc80804a) Python 3.8.1 Qt 5.15.1 PyQt 5.15.1
Platform: Linux
Flags: frz=True ao=True sv=1
Add-ons, last update check: 2022-11-03 17:55:48

Caught exception:
Traceback (most recent call last):
  File "aqt/addons.py", line 893, in onInstallFiles
  File "aqt/addons.py", line 1595, in installAddonPackages
  File "aqt/addons.py", line 467, in processPackages
  File "aqt/addons.py", line 402, in install
  File "aqt/addons.py", line 439, in _install
  File "zipfile.py", line 1628, in extract
  File "zipfile.py", line 1698, in _extract_member
  File "zipfile.py", line 1569, in open
  File "zipfile.py", line 817, in __init__
  File "zipfile.py", line 718, in _get_decompressor
  File "zipfile.py", line 695, in _check_compression
RuntimeError: Compression requires the (missing) lzma module
```

I'm running anki-2.1.49-linux which I downloaded from the anki site.

I'm not that experienced with python versions but I suspect my python setup is missing some lzma module. I tried googling around but don't know how I could add it - I don't seem to have python 3.8.1 on my system so maybe it came with the anki download.

**Comment by Ben-Kerman** [comment]
Pre-packaged releases of Anki come with a custom build of Python, and the build included in the 2.1.49 Linux release is missing the standard library's `lzma` module

Since addon files are just ZIP archives with a different extension you can create a directory in `~/.local/share/Anki2/addons21/` and extract the archive into it. Then you'll need to apply [these patches](https://github.com/Ben-Kerman/anki-jrp/issues/2#issuecomment-1190740957) in order for the addon to work properly with lzma-less Python. Finally, restart Anki and the addon should work

**Comment by sp3ctum** [comment]
I appreciate the response. I was able to install the addon with those instructions and patches 👍🏻
